plugins/inputs/qbittorrent/qbittorrent.go

```diff
@@ -0,0 +1,222 @@
+package qbittorrent
+
+import (
+	"bytes"
+	_ "embed"
+	"encoding/json"
+	"fmt"
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/config"
+	"github.com/influxdata/telegraf/plugins/inputs"
+	"io"
+	"mime/multipart"
+	"net/http"
+	"net/url"
+	"strconv"
+	"strings"
+)
+
+//go:embed sample.conf
+var sampleConfig string
+
+var globalMainData *MainData
```

Comment:
Don't have this in a global var, but as a field on the struct, otherwise it would not be possible to have 2 instances of this plugin in 1 telegraf.

Reply:
I want to confirm that the life cycle of the plug-in is consistent with the `QBittorrent`struct? I will fix it.

Reply:
For each configured input, telegraf will create a separate instance of the struct. Is that clear or what you ask?

Reply:
ok thank you. Its clear.